### ISSUE TITLE
Can we include ABS statistical areas

### ISSUE THREAD

**Issue by richardbeare** [HughParsonage/PSMA#11]
Hi,
I notice there's already a request about updating the data.

I'm wondering whether we can include some additional columns, namely references to statistical area, to allow the sort of sampling that we're currently doing by postcode on stats areas instead. I gather this data is already in G-NAF.

**Comment by HughParsonage** [comment]
From what I can see, "only" the mesh-blocks are included in the G-NAF. I use scare quotes because one can infer the others from the mesh block, but is that what you meant? The one hesitation is that they date to 2016, and the 2021 blocks are imminent.

**Comment by richardbeare** [comment]
Yep - that's what I meant, perhaps include the tables necessary to infer the others as required. I didn't realise that the 2021 versions wasn't out yet - lets wait for that.
